edb/server/server.py

```diff
@@ -906,33 +909,63 @@ async def _restart_servers_new_addr(self, nethosts, netport):
             for host, srv in self._servers.items():
                 if host == ADMIN_PLACEHOLDER or host in nethosts:
                     servers[host] = srv
+                elif host in ['::', '0.0.0.0']:
+                    servers_to_stop_early.append(srv)
```

Comment:
Should `hosts_to_start` be synced with `servers_to_stop_early`?

Reply:
The first `if` is already checking `host in nethosts`, which is the same condition populating `hosts_to_start` - not sure if that answers the question.

Reply:
Ah, I reread the code, it's all fine.